### ISSUE TITLE
Any plans for adding texture healing support?

### ISSUE THREAD

**Issue by mrboen94** [fabrizioschiavi/pragmatapro#273]
Love the font, but I would love the option to enable texture healing for the m and i separately.

https://github.com/githubnext/monaspace/blob/main/docs/Texture%20Healing.md

I have no idea how much work this is to implement in a font, but I would love for this to be optionally added to the font, as it doesn't fit everywhere.

**Comment by fabrizioschiavi** [comment]
Yes, absolutely. This solution is really genial and must be included in the next realease. Thanks!

**Comment by fabrizioschiavi** [comment]
I'm working almost full time to this improvement, what do you think of the result?

 ![PragmataPro_texture_healing](https://github.com/fabrizioschiavi/pragmatapro/assets/1576663/577dfa05-99c5-4af5-b910-c42ae69b607c)

**Comment by pkazmier** [comment]
Looking good. I am curious how a word such as “minimum” renders. Would it contain two different sized m’s? Big and small in same word?

**Comment by fabrizioschiavi** [comment]
I completely agree with you, it's a delicate balance to maintain, but it undeniably works! I rely on it daily, and I can confidently affirm that this solution has significantly enhanced readability.

At its minimum size, some imbalance becomes noticeable when magnified, yet at its practical working size, everything looks excellent.

<img width="228" alt="Screenshot 2024-01-06 at 09 37 49" src="https://github.com/fabrizioschiavi/pragmatapro/assets/1576663/3ecabc02-ddd1-4a5d-973e-9722ae871416">

**Comment by pkazmier** [comment]
Very interesting. It's great hearing your feedback on how it has greatly enhanced readability. I'm looking forward to the 0.831 release.

As one without any knowledge of font design, I'm curious how do you decide which "m" to make wider in a word such as "mime". In your sample, it was the second one, why not the first? Or why not both? Would both require a 3rd version of "i" that has even smaller serifs? Or is it a practical limit to the number of combinations you'd have to create.

Thank you for your continued work on this magnificent font. It has been one of my best purchases!

**Comment by fabrizioschiavi** [comment]
Thanks @pkazmier for your appreaciations and for bringing the `mime` issue to my attention: the `m`s near `i` must have the same width

**Comment by Teggy** [comment]
Good grief, Fabrizio.  This looks awesome!  Please keep up the good work! 👍🏼

**Comment by fabrizioschiavi** [comment]
Thank you @Teggy !

**Comment by svipas** [comment]
@fabrizioschiavi I just discovered this, holy shit - it is incredible! It always felt to me that `m` was too squished in some cases, this greatly enhances readability. I can't even imagine what 0.831 release is going to be 💥

**Comment by mrboen94** [comment]
Thanks for looking into this! Havent been active on github lately, but looks great!

**Comment by fabrizioschiavi** [comment]
Done. Available since version 0.9
Thanks to all for the idea and the suggestions!

![Image](https://github.com/user-attachments/assets/9a74b6d5-8af1-4a4b-9bf6-e942ba9e12a1)
